packages/deepbook/sources/state/state.move

```diff
@@ -294,7 +298,12 @@ module deepbook::state {
             } else {
                 math::mul(fee_volume, historic_maker_fee)
             };
-            self.history.add_total_fees_collected(balances::new(0, 0, order_maker_fee));
+
+            if (!fill.expired()) {
+                self.history.add_total_fees_collected(balances::new(0, 0, order_maker_fee));
+            } else {
```

Comment:
We can just skip line 291-300 if fill is expired right?

Reply:
We still have to make that fees calculation. If it's not expired, we end up adding that number to total_fees_collected. If it is expired, we refund the user that amount of deep.